torch/_dynamo/eval_frame.py

```diff
@@ -1274,6 +1274,9 @@ def patch():
                 opt.step = disable(opt.step)
 
             opt.zero_grad = disable(opt.zero_grad)
+            opt.state_dict = disable(opt.state_dict)
+            opt.load_state_dict = disable(opt.load_state_dict)
+            opt.add_param_group = disable(opt.add_param_group)
```

Comment:
why add_param_group

Reply:
There's no need to trace it since it doesn't have any torch ops, and it's usually called externally to the code getting compiled (during model setup).